enterprise/troubleshooting.md

```diff
@@ -82,3 +82,11 @@ SSL certificate and may also already be present in your browser explaining why
 you can open Private Packagist pages in your browser without an error. Please
 see the section about `Invalid x509 keypair` for more information on how to add
 your intermediate certificates to your certificate file.
+
+#### Moving the application to a new IP/host or behind a HTTP proxy
+
+If the IP of the host changes or you move the application behind a HTTP proxy then
```

Comment:
Actually what's more important is if you change docker's config in any way to change the internal docker IPs, rather than the external IP of the host.

Reply:
ah true, thanks!